packages/events/test/EventBoundary.tests.ts

```diff
@@ -16,7 +16,7 @@ describe('EventBoundary', function ()
         container.addChild(new Graphics().drawRect(80, 0, 100, 100));
 
         const target = container.addChild(
-            new Graphics().drawRect(150, 0, 100, 100)
+            new Graphics().drawRect(150, 0, 100, 100),
```

Comment:
This seems wrong to me, comma dangle shouldn't be on function calls.

Reply:
yep this was my mistake. Fixed in [this PR](https://github.com/pixijs/eslint-config/pull/7)